bindings/rust/evmc-client/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+/* EVMC: Ethereum Client-VM Connector API.
+ * Copyright 2019 The EVMC Authors.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pub mod host;
+mod loader;
+pub mod types;
+pub use self::loader::EvmcLoaderErrorCode;
+
+use crate::loader::evmc_load_and_create;
+use crate::types::*;
+use evmc_sys as ffi;
+use std::ffi::CStr;
+
+extern "C" {
+    fn evmc_create() -> *mut ffi::evmc_vm;
```

Comment:
I think as a first step it would be better to only support dynamic loading.

Are you using this static linking in SSVM?

Reply:
I am not sure this is relative with static linking. I still use dynamic link to lib<EvmcVM>.so and the statement just tell rustc how to resolve symbol after cargo link the library.
https://github.com/second-state/rust-ssvm/blob/master/build.rs#L25
This is useful in my case that I don't need to know where is my evmc dynamic-link library (especially the target path is not always the same).

Reply:
If it does an extern import of `evmc_create` that is statically linking against an EVMC VM, IIRC, otherwise this would be a lingering reference.